respect-app-compose/src/commonMain/kotlin/world/respect/app/app/AppNavHost.kt

```diff
@@ -584,6 +598,14 @@ fun AppNavHost(
                 )
             )
         }
+        composable<SetPassword> {
```

Comment:
Naming patterns _must_ be used consistently e.g. if the CreateAccountSetPasswordScreen, CreateAccountSetPassword (as the route object), CreateAccountSetPasswordViewModel.